mmaction/models/backbones/resnet3d_slowfast.py

```diff
@@ -457,11 +457,15 @@ def forward(self, x):
             tuple[torch.Tensor]: The feature of the input
             samples extracted by the backbone.
         """
-        x_slow = x[:, :, ::self.resample_rate, :, :]
+        x_slow = nn.functional.interpolate(
+            x, scale_factor=(1.0 / self.resample_rate, 1.0, 1.0))
         x_slow = self.slow_path.conv1(x_slow)
         x_slow = self.slow_path.maxpool(x_slow)
 
-        x_fast = x[:, :, ::self.resample_rate // self.speed_ratio, :, :]
+        x_fast = nn.functional.interpolate(
+            x,
+            scale_factor=(1.0 / (self.resample_rate // self.speed_ratio), 1.0,
```

Comment:
is `//` necessary?

Reply:
By default of the paper and PySlowfast, it is unnecessary, as in all cases it can divide (32 / 4, 32 / 8, 64 /4, 64 / 8).

Reply:
should `self.resample_rate >= self.speed_ratio` be asserted?

Reply:
Yes, resample_rate should always > speed_ratio, otherwise it will slice with step 0. But I think it will occur frequently, only when user 魔改 model, and the error log: ValueError: slice step cannot be zero is easy to track.